index.html

```diff
@@ -173,126 +173,447 @@ <h2>
+                    </li>
+                  </ol>
+                </ol>
+                <aside class="note">
+                  <p>
+                    This ensures a user can leave a document that constantly attempts
+                    to lock the pointer. The document will be blocked from locking
+                    upon initial navigation or re-acquiring lock unless the user
+                    re-engages directly with the document.
+                  </p>
+                  <p>
+                    Conversely, if pointer lock is exited via {{Document/exitPointerLock()}}
+                    no <a>engagement gesture</a> is required to reenter pointer lock.
+                    This enables applications that frequently move between
+                    interaction modes, and ones that may do so based on a timer or
+                    remote network activity.
+                  </p>
+                  <p>
+                    Pointer lock is commonly combined with fullscreen [[FULLSCREEN]],
+                    and if an <a>engagement gesture</a> is used to enter fullscreen
+                    it is sufficient for a subsequent {{Element/requestPointerLock()}}.
+                  </p>
+                </aside>
+              </li>
+              <li>
+                If <a>this</a>'s <a href="https://dom.spec.whatwg.org/#concept-node-document">node document</a>'s
+                <a data-cite="html/origin.html#active-sandboxing-flag-set">active
+                sandboxing flag set</a> has the <a>sandboxed pointer lock browsing
+                context flag</a> set:
+                <ol>
+                  1. <a href="https://html.spec.whatwg.org/multipage/webappapis.html#queue-an-element-task">
+                  Queue an element task</a> on the 
+                  <a href="https://html.spec.whatwg.org/multipage/webappapis.html#user-interaction-task-source">
+                  user interaction task source</a>, given this, to perform the following steps:
+                  <ol type='i'>
+                    <li>
+                      <a href=https://dom.spec.whatwg.org/#concept-event-fire">Fire an event</a> named
+                      <a>pointerlockerror</a> at <a>this</a>'s 
+                      <a href="https://dom.spec.whatwg.org/#concept-node-document">node document</a>.
+                    </li>
+                    <li>
+                      <a>Reject</a> <a>promise</a> with a "{{SecurityError}}" {{DOMException}}.
+                    </li>
+                    <li>
+                      Return <a>promise</a>.
+                    </li>
+                  </ol>
+                </ol>
+              </li>
+              <li>
+                <dl data-dfn-for="PointerLockOptions" data-link-for=
+                "PointerLockOptions">
+                  If <a>options</a>["<a>unadjustedMovement</a>"] is true and the platform does not
+                  support <a>unadjustedMovement</a>:
```

Comment:
Is this redundant with step 4.2? Maybe this should move before step 4, and then step 4.2 could be eliminated?